index.html

```diff
@@ -46,8 +45,110 @@ <h2 class="job"> I’m a software developer</h2>
                 <a href="#"> <img src="/images/angellist.svg" alt="Angelist Icon"></a>
             </li>
         </ul>
-
       </section>
-
+      <section class="recent-work">
+        <a href="#" class="normal-button"> <img src="/images/Disabled.svg" alt="Disabled"> </a>
+        <h1 class="recent-works">My Recent Works</h1>
```

Comment:
For SEO reasons, There should be only one `<h1>` tag on your web page. As you already used it for your name in the previous section that's why you need to replace it with hx tags (h2 ... h6) 😄